package com.akiratochiro.life_and_money_api;

import org.junit.jupiter.api.Test;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.context.annotation.Import;

@SpringBootTest(properties = "jwt.secret=46N3Mn7ipRBtI+LDENQDLRM2mXOGfeamGMqAcgFS9Gc=")
@Import(TestcontainersConfiguration.class)
class LifeAndMoneyApiApplicationTests {

	@Test
	void contextLoads() {
	}
}
